freightliner_cascadia_2012_dot_10004/drivers.launch.py

```diff
@@ -94,7 +94,7 @@ def generate_launch_description():
                             'frame_id' : 'velodyne_1',
                             'device_ip' : '192.168.1.201',
                             'port' : '2368',
-                            'gps_time' : 'True'
+                            'gps_time' : 'False'
```

Comment:
Just wanted to confirm whether this change for the `gps_time` parameter in this file and in 80550's `drivers.launch.py` was required, and if so, do we know why?

Reply:
This change was a relic of debugging LiDAR message timestamps and was not needed, just reverted this